modules/apps/notification/notification-rest-impl/src/main/java/com/liferay/notification/rest/internal/resource/v1_0/NotificationQueueEntryResourceImpl.java

```diff
@@ -265,7 +265,7 @@ private NotificationQueueEntry _toNotificationQueueEntry(
 
 						NotificationHandler notificationHandler =
 							_notificationHandlerTracker.getNotificationHandler(
-								_portal.getClassName(classNameId));
+								_portal.fetchClassName(classNameId));
```

Comment:
@fabioafreitas could you please create a test for this change? In the `NotificationQueueEntryResourceTest`